j5/components/motor.py

```diff
@@ -61,6 +61,6 @@ def power(self) -> MotorState:
     def power(self, new_power: MotorState) -> None:
         """Set the current state of this output."""
         if isinstance(new_power, float):
-            if new_power < -1 or new_power > 1:
+            if not(-1 < new_power < 1):
```

Comment:
I believe this would error in the case where the power is 1 or -1
```
>>> not(-1 < 1 < 1)
True
>>> not(-1 < -1 < 1)
True
```

compared to
```
>>> 1 < -1 or 1 < 1
False
>>> -1 < -1 or -1 < 1
False
```

I'm also not a huge fan of shortening code for the sake of shortening code. Personally I have to stop and try to parse shorthand boolean logic and it takes me a while to understand intent whereas with the previous syntax its blatantly obvious to a beginner exactly what is being checked.

Reply:
Quite surprised that this wasn't caught in tests. I should rely on them less when accepting changes.

This should be tested [here](https://github.com/j5api/j5/blob/4e847333a2a93b1c710dad060f2b56553e58a1da/tests/components/test_motor.py#L86) but doesn't appear to for some reason.